### DISCUSSION TITLE
Cannot enable

### DISCUSSION THREAD

**Discussion by thphamt** [noah-nuebling/mac-mouse-fix#244] [Questions] [answered]
Hey guys, 

I'm having this problem where I cannot enable the features at all, clicking on the "Enable Mac Mouse Fix" box does nothing

![CleanShot 2022-02-16 at 8 AM 20 31@2x](https://user-images.githubusercontent.com/94689625/154178064-71bc1df9-c95e-4176-9765-1e5d4d11eea7.jpg)

Anyone else having this problem or found a solution?
Thanks

**Comment by paulter** [comment]
Hi! I've just downloaded the app for the first time and currently it's the same experience for me. I've even enabled accessibility and input monitoring rights (System Preferences => Security & Privacy) for Mac Mouse Fix, hoping that would resolve the issue, but that did not change anything. Could this possibly be a conflict with Karabiner Elements? (although I had them disabled at the moment) 

MacOS Monterey 12.2.1, MacBook Pro (13-inch, M1, 2020), Mac Mouse Fix Version 2.2.0 (6943)

**Comment by noah-nuebling** [comment, marked as answer]
Hey guys! Please have a look at [this thread](https://github.com/noah-nuebling/mac-mouse-fix/issues/192). They were having the same issue and some could even find a solution. 

Unfortunately, there can be many different reasons for this issue so it's not sure that their solution will work for you.

If you file a [Bug Report](https://noah-nuebling.github.io/mac-mouse-fix-feedback-assistant/?type=bug-report) and attach some Console Logs, that might help figure this out.

@thphamt It might very well be interference with Karabiner Elements. However if the 'Enable Mac Mouse Fix' checkbox stays disabled after you click it (like @thphamt) I'd guess it's something else.

**Reply by thphamt** [reply]
I ran this command `launchctl enable gui/$UID/mouse.fix.helper` and it worked!
Thanks so much
